ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/ethernet_interfaces.py

```diff
@@ -35,17 +35,33 @@ def ethernet_interfaces(self) -> dict | None:
                     # The l3_interfaces has already been filtered in _filtered_tenants
                     # to only contain entries with our hostname
                     for l3_interface in vrf["l3_interfaces"]:
+                        nodes_length = len(l3_interface["nodes"])
+                        if (
+                            len(l3_interface["interfaces"]) != nodes_length
+                            or len(l3_interface["ip_addresses"]) != nodes_length
+                            or ("descriptions" in l3_interface and "description" not in l3_interface and len(l3_interface["descriptions"]) != nodes_length)
+                        ):
+                            raise AristaAvdError(
+                                f"Length of interfaces, nodes and ip_addresses and descriptions (if used) must match for l3_interfaces for {vrf['name']} in"
```

Comment:
```suggestion
                                f"Length of lists 'interfaces', 'nodes', 'ip_addresses' and 'descriptions' (if used) must match for 'l3_interfaces' for {vrf['name']} in"
```

Reply:
Addressed during rebasing (yeah... that was a bit unorthodox) in c2af0a2